omnipaxos_core/src/sequence_paxos/mod.rs

```diff
@@ -346,7 +346,7 @@ where
             return;
         } else if pid == self.leader.pid {
             self.state = (Role::Follower, Phase::Recover);
-            self.current_seq_num = 0;
+            self.current_seq_num.next_round();
```

Comment:
How about we keep this and use the fact that we have incremented the `round` (`session`) to allow the follower to avoid sending multiple `PrepareReq`. So if the follower gets (1,2) (1,4) (1,5) only receiving (1,4) it will go to (2,0) and then (1,5) can be ignored.

Reply:
I think (1,5) will be ignored anyway in `handle_acceptdecide` because it is in the `Recover` state

Reply:
The point is also that I think it is better if only the leader sets the seq numbers and followers don't touch it and only check if it got the expected message.